HACKING.md

```diff
@@ -327,22 +327,22 @@ To make a release of the `ops` and/or `ops-scenario` packages, do the following:
 2. Visit the [releases page on GitHub](https://github.com/canonical/operator/releases).
 3. Click "Draft a new release"
 4. The "Release Title" is the full version numbers of ops and/or ops-scenario,
-   in the form `ops <major>.<minor>.<patch> and ops-scenario <major>.<minor>.<patch>`
+   in the form `<major>.<minor>.<patch> and ops[testing] <major>.<minor>.<patch>`
```

Comment:
The catch here is that the package we are releasing is called `ops` or `ops-scenario`. Using "ops[testing]" here seems misleading.

In terms of leaving "ops" out when we're releasing both (or, I assume, only ops), I'm reluctantly on board with that, given that "ops" is the thing that brings together what's in this repo, and the most expected package.

Reply:
Yeah, for this I'd probably stick with what we have given these are the PyPI package names.

Reply:
If both ops and ops-scenario are being released (which should usually be the case, right?) then shall we go with option 1 or option 2:
1. `ops <major>.<minor>.<patch> and ops-scenario <major>.<minor>.<patch>`
2. `<major>.<minor>.<patch> and ops-scenario <major>.<minor>.<patch>` 

The difference being whether we mention `ops` at the beginning

Reply:
Decision is to go with option 1 for clarity.